tests/testthat/test-admiral_options.R

```diff
@@ -35,4 +35,26 @@ test_that("set_admiral_options Test 5: unexpected function input for set gives e
     class = "assert-admiraldev"
   )
 })
+
+## Test 6: non-positive numeric argument triggers assertion error ----
+test_that("get_admiral_option Test 6: non-positive numeric argument triggers assertion error", {
+  sigfigs <- 0
+  expect_error(
+    set_admiral_options(signif_digits = sigfigs),
+    class = "assert_integer_scalar"
+  )
+  sigfigs <- -1
+  expect_error(
+    set_admiral_options(signif_digits = sigfigs),
+    class = "assert_integer_scalar"
+  )
+})
+
+## Test 7: set works ----
```

Comment:
Minor comment: please update the title of Test 7 and Test 4 to be a bit more descriptive, i.e. specify which option you are testing?